README.md

```diff
@@ -126,6 +126,18 @@ You can optionally specify the start and end dates to backtest over a specific t
 poetry run python src/backtester.py --ticker AAPL,MSFT,NVDA --start-date 2024-01-01 --end-date 2024-03-01
 ```
 
+## Using Docker
+
```

Comment:
Add the info to create a .env file first.

Reply:
.env is assumed to exist on localhost (it's .gitignored)